README.md

```diff
@@ -126,6 +126,14 @@ class TestElement extends Polymer.Element {
   @property()
   bar: string = 'yes';
 
+  // @observe does not support simple observers,
+  // you can already do that via a normal property
+  @property({ observer: TestElement.prototype.onBazChanged })
```

Comment:
This depends on https://github.com/Polymer/polymer-decorators/pull/18 right? That requires that `observer` is a `string`, which is also how Polymer normally works (AFAIK you can't pass a function reference in the normal Polymer block, or at least that's an undocumented feature if you can).

Reply:
It depends on #18 for the type definitions (`PropertyOptions` has no `observer` in master).

You can pass a function reference as of very recently due to polymer/polymer#4574. it doesn't seem to be documented well, if at all.

its probably best we recommend this, though, as its a little stronger typed than the string version.

Reply:
> It depends on #18 (PropertyOptions has no observer in master). so yes we need to change it to string|((val: any, old: any) => void) or something i suppose, maybe in this PR?
> 
> You can pass a function reference as of very recently due to polymer/polymer#4574. it doesn't seem to be documented well, if at all.

Ah, I missed that change. Yes, would be great to also add the new signature to this PR, `string|((val: any, old: any) => void)` seems right.

BTW we should also update https://github.com/Polymer/polymer/blob/master/types/extra-types.d.ts to match https://github.com/Polymer/polymer/blob/master/externs/polymer-externs.js#L24, maybe in https://github.com/Polymer/polymer/pull/5000?

Reply:
yup can do. i added the `observer` definition here too now.